manager/config/crd/bases/app.m4d.ibm.com_m4dapplications.yaml

```diff
@@ -78,11 +84,13 @@ spec:
                     required:
                     - protocol
                     type: object
+                  recurrence:
```

Comment:
I suggest we move this to the catalog as metadata, I don't think the manager should be aware of this. Only the code that interacts with the "copy module" should be aware of this.

Reply:
@simanadler ^

Reply:
We could move the recurrence property to the inline catalog as a part of dataset metadata. However, if there is a request to read data from the enterprise catalog and refresh with some frequency, do we have a way to do this? Can any metadata be inserted to the catalog? I guess not.

Reply:
@shlomitk1 I think that scenario is out of scope for our initial implementation.
But generally catalogs support metadata, which can include refresh frequency, and we cab get it from there.

Reply:
OK, removing the recurrence definition from m4dapplication.